scss/components/_sidebar-nav.scss

```diff
@@ -10,13 +10,77 @@
         padding: 0;
         margin: 0;
         list-style: none;
+
+        a {
+            //padding-left: 68px;
+        }
+        .sidebar-nav__opener {
+            padding-left: 68px;
+        }
+        .sidebar-nav__link {
+        }
+
+        [class*=icon-] {
+            left: 28px;
+        }
+        ul {
+             a {
+                //padding-left: 84px;
+            }
+            .sidebar-nav__opener {
+                padding-left: 84px;
+            }
+            .sidebar-nav__link {
+                padding-left: 68px;
+            }
+            [class*=icon-] {
+                left: 44px;
+            }
+        }
+        ul ul {
+            a {
+                //padding-left: 100px;
+            }
+            .sidebar-nav__opener {
+                padding-left: 100px;
+            }
+            .sidebar-nav__link {
+                padding-left: 84px;
+            }
+            [class*=icon-] {
+                left: 60px;
+            }
+        }
+        ul ul ul {
+            .sidebar-nav__opener {
+                padding-left: 116px;
+            }
+            .sidebar-nav__link {
+                padding-left: 100px;
+            }
+            [class*=icon-] {
+                left: 76px;
+            }
+        }
+        ul ul ul ul {
```

Comment:
Why we cann't add paddings to `ul`'s? Or in the worst case create a mixin to generate all paddings.